caom2pipe/caom_composable.py

```diff
@@ -1136,6 +1135,32 @@ def update(self, file_info):
         self._logger.debug('End update')
         return self._observation
 
+    def _init_read_groups(self, config):
+        for entry in config.data_read_groups:
+            self._data_read_groups.add(entry)
+        for entry in config.meta_read_groups:
+            self._meta_read_groups.add(entry)
+
+    def _update_artifact(self, artifact):
+        """
+        :param artifact: Artifact instance
+        :return:
+        """
+        return
```

Comment:
Was it intentional to leave it noop?

Reply:
Replaced with `raise NotImplementedError`